package.json

```diff
@@ -81,7 +81,7 @@
     "audit-ci": "^5.1.2",
     "browserfs": "^1.4.3",
     "danger": "^11.2.0",
-    "eslint": "^8.9.0",
+    "eslint": "^8.30.0",
```

Comment:
I updated it because the logical assignment operator rule was introduced in ESLint v8.24.0.
https://eslint.org/docs/latest/rules/logical-assignment-operators#version